manifest-celery.yml.j2

```diff
@@ -0,0 +1,29 @@
+---
+applications:
+- name: notify-template-preview-celery
+
+  disk_quota: 2G
+  memory: 2G
+  health-check-type: process
+  command: exec scripts/run_app_paas.sh celery -A run_celery.notify_celery worker --loglevel=INFO --concurrency=5 --uid=`id -u celeryuser`
+
+  services:
+    - logit-ssl-syslog-drain
+
+  routes:
+    - route: {{ environment }}-notify-template-preview-celery.cloudapps.digital
```

Comment:
I think the reason that template-preview is the only app to have the environment at the beginning of the route is to avoid routes called `notify-template-preview-preview` or `notify-template-preview-production` which might be a bit unclear.

Keeping the route as `preview-notify-template-preview-celery` seems maybe(?) a bit clearer than `notify-template-preview-celery-preview`.